### DISCUSSION TITLE
分类页相同设置下内容不一致

### DISCUSSION THREAD

**Discussion by Yuri-YuzuChaN** [xiaoyaocz/flutter_dmzj#91] [General]
设置为 更新排序，ゆり题材，地域日本，原版dmzj跟旧版是一致的
![Screenshot_2023-03-15-01-33-16-051_com miui home](https://user-images.githubusercontent.com/65105396/225091861-8add875b-0af3-49a2-b4e2-4536e20f461f.jpg)

**Comment by oubeichen** [comment]
应该是bug把第一页内容漏了

**Comment by xiaoyaocz** [comment]
页数是从0开始，而不是1😂
[已修复](https://github.com/xiaoyaocz/flutter_dmzj/commit/d74b695bfe8c4b8c75f391470db83103714ae1f0)
